osism/core/enums.py

```diff
@@ -110,6 +110,7 @@
     "ulimits": {"environment": "generic", "runtime": "osism-ansible"},
     "mariadb-backup": {"environment": "kolla", "runtime": "kolla-ansible"},
     "mariadb-recovery": {"environment": "kolla", "runtime": "kolla-ansible"},
+    "tempest": {"environment": "openstack", "runtime": "osism-ansible"},
```

Comment:
This is not necessary. You have to add a playbook openstack/tempest.yml in the osism/ansible-playbooks repository. Like for the osism.validations.refstack role: https://github.com/osism/ansible-playbooks/blob/main/playbooks/openstack/refstack.yml

Reply:
Got it, [the `validate-tempest.yml` symlink is sufficient](https://github.com/osism/python-osism/blob/main/osism/commands/validate.py#L108)